### PR TITLE
added functions for showing and hiding elements so that tags can be seen more clearly.

### PR DESCRIPTION
**Pull request by seanmcguire12**

### CODE REVIEW

File: tarsier/tag_utils.ts [hunk 1]
```diff
@@ -139,7 +141,7 @@ function create_tagged_span(idNum: number, el: HTMLElement) {
   idSpan.style.position = "relative";
   idSpan.style.display = "inline";
   idSpan.style.color = "white";
-  idSpan.style.backgroundColor = "red";
+  idSpan.style.backgroundColor = "blue";
```
Comment (asim-shrestha) [inline tarsier/tag_utils.ts:144]: Let's revert this now :)

File: tarsier/tag_utils.ts [hunk 2]
```diff
@@ -386,3 +388,36 @@ const showMapElements = () => {
...
+
+window.hideElements = () => {
+  const allElements = document.body.querySelectorAll("*");
+  allElements.forEach((el) => {
+    const element = el as HTMLElement;
+
+    // if the element has the visibility property, then copy the value of that property into
+    // a new attribute called 'reworkd-original-visibility'
+    if (element.style.visibility){
+      element.setAttribute('reworkd-original-visibility', element.style.visibility);
```
Comment (asim-shrestha) [inline tarsier/tag_utils.ts:400]: let's store this string in a variable so that we can reference it in both places

File: tarsier/tag_utils.ts [hunk 3]
```diff
@@ -386,3 +388,36 @@ const showMapElements = () => {
     (element as HTMLElement).style.opacity = element.getAttribute('data-original-opacity') || '1';
   });
 }
+
+window.hideElements = () => {
```
Comment (asim-shrestha) [inline tarsier/tag_utils.ts:392]: Might make sense to update naming here

```suggestion
window.hideNonTagElements = () => {
```

**Comment by awtkns** [comment]
Make sure to prefix you commit messages an emoji 😉 

[some inspiration](https://gitmoji.dev/)

**Pull request merged**